src/config/index.ts

```diff
@@ -172,3 +176,40 @@ export const getExplorerInfo = (hash: string): BlockScanInfo => {
 
   return blockScanInfo
 }
+
+// With some wallets from web3connect you have to use their provider instance only for signing
```

Comment:
regarding your concern about moving this block of code to this file, a simple solution could be to move this to another file and export it as a function that receives the rpcUrl.

Reply:
perhaps, a better solution is to write it as a hook.